app/assets/stylesheets/fortitude/utilities/_responsive-width.scss

```diff
@@ -14,13 +14,18 @@ Utility                | Description
 `of`                   | 1`of`12 meaning you know you'll get a percent and not rhyhtm unit.
 `12`                   | column total (default is 12)
 
-```html_example
-<pre class="xs-6of12">
-  width: 50% for all screen sizes
-</pre>
-<pre class="md-6of12">
-  50% width for medium screens and up
-</pre>
+```html_preview_example
+<div class="xs-6of12">
+  <div class="box box--styleguide">
+    50% width for all screen sizes
```

Comment:
In general, I left it out for English sentences, but put it in for demonstrative css output / other code. We should probably decide a consistent style there at some point, but I'd rate it non-critical.